components/brave_shields/browser/tracking_protection_service.cc

```diff
@@ -20,10 +20,31 @@
 #include "brave/components/brave_shields/browser/dat_file_util.h"
 #include "brave/vendor/tracking-protection/TPParser.h"
 
-#define DAT_FILE "TrackingProtection.dat"
+#if BUILDFLAG(BRAVE_STP_ENABLED)
+#include "base/strings/string_split.h"
+#include "base/task/post_task.h"
+#include "brave/common/brave_switches.h"
+#include "brave/components/brave_shields/browser/brave_shields_util.h"
+#include "brave/components/brave_shields/browser/brave_shields_web_contents_observer.h"
+#include "brave/components/brave_shields/common/brave_shield_constants.h"
+#include "components/content_settings/core/browser/host_content_settings_map.h"
+#include "content/browser/web_contents/web_contents_impl.h"
+#include "content/public/browser/web_contents.h"
+#include "content/public/browser/browser_task_traits.h"
```

Comment:
I'd suggest to apply clang-format (at least, to newly added files)

Reply:
Not sure why `npm run lint` did not pick it up

Reply:
liter doesn't check formatting and other stuff like sorting of #include